src/tools/rbd_mirror/PoolReplayer.h

```diff
@@ -16,12 +22,6 @@
 #include "ImageDeleter.h"
 #include "types.h"
 
-#include <set>
```

Comment:
@chardan If it's already in master, it shouldn't show up as a change in this PR.